### ISSUE TITLE
Remove the list comprehension stuff from the shape measurements...

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#668]
...I had a feeling this was too complicated.

**Comment by manerotoni** [comment]
Given that there are only few objects a pandas list and browsing through it is perfectly fine. You do not want to teach list comprehension

**Comment by tischi** [comment]
Yes, in fact we discussed after the course that the best may be to immediately convert the output of the shape measurements to a pandas dataframe an access the results like this. This would allow us to remove _a lot_ of (rather awkward) code for accessing the regionprops directly. Would you agree with this?
